ui/cap-react/src/antd/schemas/Schemas.js

```diff
@@ -0,0 +1,263 @@
+              typeof e[1] === "object" && e[1] !== null && !Array.isArray(e[1])
+          )
+          .map(e => ({
+            label: e[0],
+            value: e[0],
+          }));
+        opts.unshift({ label: FULL_SCHEMA, value: "all" });
+        setOptions(opts);
+      })
+      .catch(() => {
+        setError(true);
+      });
+  }, []);
+
+  const handleSelectChange = (value, option) => {
+    if (value === "all") {
+      setField(schema);
+    } else {
+      setField(schema[value]);
+    }
+    setSelection(option.label);
+  };
+
+  const handleEdit = value => {
+    if (!value.error) {
+      const jsObject = value.jsObject;
+      if (selection === FULL_SCHEMA) {
+        setSchema(jsObject);
+      } else {
+        setSchema({ ...schema, [selection]: jsObject });
+      }
+      setField(jsObject);
+    }
+    setJsonError(value.error);
+  };
+
+  const handleSave = () => {
+    let schemaToSave = schema;
+    if (selection != FULL_SCHEMA) {
+      schemaToSave = { ...originalSchema, [selection]: schema[selection] };
+    }
+    axios
+      .put(`/api/jsonschemas/${schema_name}/${schema.version}`, schemaToSave)
+      .then(() => {
+        setOriginalSchema(schema);
+        notification.success({
+          message: "Schema updated",
+          description: "Changes successfully applied",
+        });
+      })
+      .catch(() =>
+        notification.error({
+          message: "Schema not updated",
+          description: "Error while saving, please try again",
+        })
+      );
+  };
+
+  const handleRevert = () => {
+    if (selection === FULL_SCHEMA) {
+      setField(originalSchema);
+      setSchema(originalSchema);
+    } else {
+      setField(originalSchema[selection]);
+      setSchema({ ...schema, [selection]: originalSchema[selection] });
+    }
+  };
+
+  return (
+    <DocumentTitle title="Schemas">
+      {error ? (
+        <ErrorScreen />
+      ) : (
+        <React.Fragment>
+          <Modal
+            visible={showModal}
+            onCancel={() => setShowModal(false)}
+            title={`${selection} diff`}
+            width={800}
+            footer={null}
+            bodyStyle={{
+              overflowX: "scroll",
+              overflowY: "auto",
+              maxHeight: "calc(100vh - 200px)",
+            }}
+          >
+            <JsonDiff
+              left={
+                selection === FULL_SCHEMA
+                  ? originalSchema
+                  : originalSchema[selection]
+              }
+              right={field}
+            />
+          </Modal>
+          <Row justify="center">
+            <Col xs={22} lg={14}>
+              <Row
+                justify="space-between"
+                align="middle"
+                style={{ marginBottom: "1em" }}
+                gutter={10}
+              >
+                <Col flex="auto">
+                  <Typography.Title style={{ marginBottom: "0" }}>
+                    {schema_name} {schema_version}
```

Comment:
`schema_version` should be taken from the response. when we pass `/schemas/cms-analysis` without a version, the API returns the latest schema version for this content-type => version number is on response. Lets display this one